### ISSUE TITLE
This isnt your code to release.

### ISSUE THREAD

**Issue by GavinDarkglider** [g91/FreestyleDashAurora#1]
You dont have the right to release this code, publically or otherwise. This code is the property of TeamFSD, or now known as Team Phoenix. Please remove this repo, or at least make it private.

**Comment by StelioKontosXBL** [comment]
On that note, the lack of a license in the contained source code, which is the property of Team Phoenix as stated above, is legally considered a full copyright. Meaning any usage of this code, in any project whether open or closed source, will make that user liable for violating copyright and intellectual property theft laws, including @g91 for his involvement by publishing someone else's copyright work.

All of this carries with it serious legal ramifications that are less than desirable especially for someone in your current legal situation, @g91.

**Comment by TempAccountNull** [comment]
On that note, the lack of a license in the contained source code, which is the property of Team Phoenix as stated above, is legally considered a full copyright. Meaning any usage of this code, in any project whether open or closed source, will make that user liable for violating copyright and intellectual property theft laws, including @g91 for his involvement by publishing someone else's copyright work indeed.


I don't know how you managed to get a hold of this, but, I'm having a DMCA filed against you.

**Comment by g91** [comment]
you're right but the company has to be a registered liable company. and they are not registered nor have the ability to sue. and I'm sorry if that upsets you but the source codes going to stay up and it's been published other places.
